_src/_posts/2017-01-31-learning-rtb.markdown

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title:  Learning from the RTB market
+date:   2017-01-31
+tag: data-science
+categories:
+  - data-science
+keywords: "python, RTB, machine-learning"
+author: memeplex
+---
+
+As you might already know, [Jampp][jampp] is a demand-side platform ([DSP][dsp])
+that allows companies to promote their mobile applications by leveraging
+real-time bidding ([RTB][rtb]) technologies in order to buy digital advertising
+from multiple inventory sources and exchanges. During an RTB transaction, an
+auction is announced and any interested bidder has to answer with a bid price
+within a time constraint of about 100 milliseconds. The bidder that wins the
+auction pays the second-highest price and obtains the right to print a creative
+(which is just jargon for displaying an ad) on a publisher site. If the banner
+is clicked by the user, further goal events might be tracked, v.g. the
+application may be installed or opened. Obtaining a goal event is called a
+conversion.
+
+One main component of our platform is the machine learning subsystem that feeds
+the bidding subsystem with predictive models of conversion rates for different
+events of interest (clicks, installs, opens, etc.). A conversion rate for an
+event $E$ is the conditional probability $p(E|C,T)$ of getting that event by
+printing some creative $C$ in the context of the current RTB transaction $T$.
+Why is it so important for our business to have good estimates of these rates?
+Simply because we sell conversions but we pay for impressions. For example,
+suppose one client pays us $\$1$ per install and we estimate the install
+conversion rate to be around $0.1$ (just cheating... real world figures are well
+under a dismaying $0.001$ `:(`). Then, assuming our creative is going to get
+printed, we should expect an average profit of $\$0.1 - c$, where $c$ is the
+expected cost of the impression. If our rate estimates are grossly miscalculated
+we will be losing money by valuating the expected income too low (thus missing
+opportunities with $c < \$0.1$) or too high (thus buying non-opportunities with
```

Comment:
This sentence is hard to follow: "$c" is written as a variable, but in the example it is supposed to be fixed in some value (say 0.2). The conversion rate is fixed in $0.1$, but it actually depends on the estimations, and should be a variable ("$cvr", for instance). If "valuating the expected income too low " takes place, it's because "$cvr" is too low. But in the example it looks as a fixed 0.1 value.